onnxruntime/core/providers/opencl/opencl_provider_factory.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/opencl/opencl_provider_factory.h"
+#include "core/framework/execution_provider.h"
+#include "core/session/abi_session_options_impl.h"
+#include "opencl_execution_provider.h"
+
+#include <atomic>
+
+namespace onnxruntime {
+struct OpenCLExecutionProviderFactory final : IExecutionProviderFactory {
+  OpenCLExecutionProviderInfo info;
+
+  OpenCLExecutionProviderFactory() = default;
+  ~OpenCLExecutionProviderFactory() final = default ;
```

Comment:
nit: please run clang-format over all files. that should have removed the space between the `default` and trailing `;`

maybe do that as a final step once everything else is fixed so we're not mixing diffs from formatting with real changes.

Reply:
Done